File: converter-dotnet-core-console-cs/MhtmlFileConversion.cs
using ceTe.DynamicPDF.Conversion;
using System;
using System.Collections.Generic;
using System.Text;

namespace converter_dotnet_core_console_cs
{
    public class MhtmlFileConversion
    {
        public static void Run(string outputPdf)
        {
            Converter.Convert(Program.GetResourcePath(@"DocumentA.mht"), Program.GetOutputDocPath(outputPdf));
            Console.WriteLine("MhtmlFileConversion Example Executed.");
        }
    }
}

File: converter-dotnet-core-console-vb/MhtmlFileConversion.vb
Imports ceTe.DynamicPDF.Conversion

Public Class MhtmlFileConversion
    Public Shared Sub Run(outputPdf As String)

        Converter.Convert(Program.GetResourcePath("DocumentA.mht"), Program.GetOutputDocPath(outputPdf))
        Console.WriteLine("MhtmlFileConversion Example Executed.")

    End Sub
End Class
